diff --git a/node_modules/react-native-google-fit/android/build.gradle b/node_modules/react-native-google-fit/android/build.gradle
index 8835c6a..4e5155e 100755
--- a/node_modules/react-native-google-fit/android/build.gradle
+++ b/node_modules/react-native-google-fit/android/build.gradle
@@ -3,7 +3,6 @@ buildscript {
         google()
         mavenLocal()
         mavenCentral()
-        jcenter()
     }
 
     dependencies {
@@ -62,7 +61,7 @@ android {
 repositories {
     google()
     mavenLocal()
-    jcenter()
+    mavenCentral()
     maven {
         // All of React Native (JS, Obj-C sources, Android binaries) is installed from npm
         url "$rootDir/../node_modules/react-native/android"
